Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/MicrophoneInput.cs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 23da1ca86bbacd443bf292961021ab9f
+guid: e66f45057da5af545a9d806866580125
```

Comment:
I've noticed almost all of the scripts have been re-GUIDed in their meta files. Was that intentional? It may break prefab and scene connections

Reply:
Yeah this is not ideal at all.

Reply:
Yeah this is not ideal at all.

Reply:
It seems like something strange happened here when merging the feature/spectator view payload into my local branch. I'll revert and reattempt this merge when I get back to my desk

Reply:
It seems like we may be working in different unity versions. I've kept the same guids now, but some parameters in the meta file have been removed. For example, theres no longer anything indicating the unity license used.